src/test/scala/features/stats/spark/StatsGeneratorTestBase.scala

```diff
@@ -33,6 +33,7 @@ abstract class StatsGeneratorTestBase extends FunSuite with BeforeAndAfterAll {
                                           .appName(appName)
                                           .enableHiveSupport()
                                           .config("spark.driver.memory", "1.5g")
+                                          .config("spark.sql.warehouse.dir", "target/spark_warehouse" )
```

Comment:
This will not do what you expect if the current working directory isn't the root of the project.